tests/integration/test_ourlogs.py

```diff
@@ -174,32 +164,66 @@ def test_ourlog_extraction_with_sentry_logs(
         "body": "Example log record",
         "span_id": "eee19b7ec3c1b174",
         "severity_text": "info",
-        "severity_number": 10,
+        "severity_number": 9,
         "attributes": {
             "boolean.attribute": {"bool_value": True},
             "sentry.severity_number": {"int_value": 9},
             "sentry.severity_text": {"string_value": "info"},
         },
     }
-    expected_with_missing_fields = {
+
+    del ourlogs[0]["received"]
+    assert ourlogs[0] == expected
+
+    ourlogs_consumer.assert_empty()
+
+
+def test_ourlog_extraction_with_sentry_logs_with_missing_fields(
+    mini_sentry,
+    relay_with_processing,
+    ourlogs_consumer,
+):
+
+    ourlogs_consumer = ourlogs_consumer()
+    project_id = 42
+    project_config = mini_sentry.add_full_project_config(project_id)
+    project_config["config"]["features"] = [
+        "organizations:ourlogs-ingestion",
+    ]
+
+    relay = relay_with_processing(options=TEST_CONFIG)
+
+    start = datetime.now(timezone.utc)
+
+    envelope = envelope_with_sentry_logs(
+        {
+            "timestamp": start.timestamp(),
+            "trace_id": "5b8efff798038103d269b633813fc60c",
+            "level": "warn",
+            "body": "Example log record 2",
+        }
+    )
+    relay.send_envelope(project_id, envelope)
+
+    ourlogs = ourlogs_consumer.get_ourlogs()
+    assert len(ourlogs) == 1
+    expected = {
         "organization_id": 1,
         "project_id": 42,
         "retention_days": 90,
         "timestamp_nanos": time_within_delta(start, expect_resolution="ns"),
         "observed_timestamp_nanos": time_within_delta(start, expect_resolution="ns"),
         "trace_id": "5b8efff798038103d269b633813fc60c",
-        "body": "Example log record",
+        "body": "Example log record 2",
         "severity_text": "warn",
         "severity_number": 13,
+        "attributes": {
+            "sentry.severity_number": {"int_value": 13},
+            "sentry.severity_text": {"string_value": "warn"},
+        },
     }
-
     del ourlogs[0]["received"]
-    assert ourlogs[0] == expected_with_all_fields
-    del ourlogs[1]["received"]
-    del ourlogs[1]["span_id"]
-    assert ourlogs[1] == expected_with_missing_fields
-
-    ourlogs_consumer.assert_empty()
+    assert ourlogs[0] == expected
```

Comment:
Same here, `assert ourlogs = [expected]` and you don't even have to assert the length.